--- ../src-base/minecraft/net/minecraft/client/resources/IResource.java
+++ ../src-work/minecraft/net/minecraft/client/resources/IResource.java
@@ -4,10 +4,7 @@
 import java.io.InputStream;
 import net.minecraft.client.resources.data.IMetadataSection;
 import net.minecraft.util.ResourceLocation;
-import net.minecraftforge.fml.relauncher.Side;
-import net.minecraftforge.fml.relauncher.SideOnly;
 
-@SideOnly(Side.CLIENT)
 public interface IResource extends Closeable
 {
     ResourceLocation func_177241_a();
