--- ../src-base/minecraft/net/minecraft/client/resources/Locale.java
+++ ../src-work/minecraft/net/minecraft/client/resources/Locale.java
@@ -10,12 +10,9 @@
 import java.util.Map;
 import java.util.regex.Pattern;
 import net.minecraft.util.ResourceLocation;
-import net.minecraftforge.fml.relauncher.Side;
-import net.minecraftforge.fml.relauncher.SideOnly;
 import org.apache.commons.io.Charsets;
 import org.apache.commons.io.IOUtils;
 
-@SideOnly(Side.CLIENT)
 public class Locale
 {
     private static final Splitter field_135030_b = Splitter.on('=').limit(2);
@@ -95,6 +92,8 @@
 
     private void func_135021_a(InputStream p_135021_1_) throws IOException
     {
+        p_135021_1_ = net.minecraftforge.fml.common.FMLCommonHandler.instance().loadLanguage(field_135032_a, p_135021_1_);
+        if (p_135021_1_ == null) return;
         for (String s : IOUtils.readLines(p_135021_1_, Charsets.UTF_8))
         {
             if (!s.isEmpty() && s.charAt(0) != 35)
