--- ../src-base/minecraft/net/minecraft/client/resources/SimpleResource.java
+++ ../src-work/minecraft/net/minecraft/client/resources/SimpleResource.java
@@ -13,11 +13,8 @@
 import net.minecraft.client.resources.data.IMetadataSection;
 import net.minecraft.client.resources.data.IMetadataSerializer;
 import net.minecraft.util.ResourceLocation;
-import net.minecraftforge.fml.relauncher.Side;
-import net.minecraftforge.fml.relauncher.SideOnly;
 import org.apache.commons.io.IOUtils;
 
-@SideOnly(Side.CLIENT)
 public class SimpleResource implements IResource, Closeable
 {
     private final Map<String, IMetadataSection> field_110535_a = Maps.<String, IMetadataSection>newHashMap();
