--- ../src-base/minecraft/net/minecraft/client/resources/data/LanguageMetadataSection.java
+++ ../src-work/minecraft/net/minecraft/client/resources/data/LanguageMetadataSection.java
@@ -5,7 +5,6 @@
 import net.minecraftforge.fml.relauncher.Side;
 import net.minecraftforge.fml.relauncher.SideOnly;
 
-@SideOnly(Side.CLIENT)
 public class LanguageMetadataSection implements IMetadataSection
 {
     private final Collection<Language> field_135019_a;
