--- ../src-base/minecraft/net/minecraft/client/resources/data/LanguageMetadataSectionSerializer.java
+++ ../src-work/minecraft/net/minecraft/client/resources/data/LanguageMetadataSectionSerializer.java
@@ -10,10 +10,7 @@
 import java.util.Map.Entry;
 import net.minecraft.client.resources.Language;
 import net.minecraft.util.JsonUtils;
-import net.minecraftforge.fml.relauncher.Side;
-import net.minecraftforge.fml.relauncher.SideOnly;
 
-@SideOnly(Side.CLIENT)
 public class LanguageMetadataSectionSerializer extends BaseMetadataSectionSerializer<LanguageMetadataSection>
 {
     public LanguageMetadataSection deserialize(JsonElement p_deserialize_1_, Type p_deserialize_2_, JsonDeserializationContext p_deserialize_3_) throws JsonParseException
