--- ../src-base/minecraft/net/minecraft/client/resources/data/PackMetadataSection.java
+++ ../src-work/minecraft/net/minecraft/client/resources/data/PackMetadataSection.java
@@ -4,7 +4,6 @@
 import net.minecraftforge.fml.relauncher.Side;
 import net.minecraftforge.fml.relauncher.SideOnly;
 
-@SideOnly(Side.CLIENT)
 public class PackMetadataSection implements IMetadataSection
 {
     private final ITextComponent field_110464_a;
