--- ../src-base/minecraft/net/minecraft/client/resources/data/PackMetadataSectionSerializer.java
+++ ../src-work/minecraft/net/minecraft/client/resources/data/PackMetadataSectionSerializer.java
@@ -9,10 +9,7 @@
 import java.lang.reflect.Type;
 import net.minecraft.util.JsonUtils;
 import net.minecraft.util.text.ITextComponent;
-import net.minecraftforge.fml.relauncher.Side;
-import net.minecraftforge.fml.relauncher.SideOnly;
 
-@SideOnly(Side.CLIENT)
 public class PackMetadataSectionSerializer extends BaseMetadataSectionSerializer<PackMetadataSection> implements JsonSerializer<PackMetadataSection>
 {
     public PackMetadataSection deserialize(JsonElement p_deserialize_1_, Type p_deserialize_2_, JsonDeserializationContext p_deserialize_3_) throws JsonParseException
